### PR TITLE
Update ts0601 add new class

### PR DESCRIPTION
**Pull request by caslubas**
Add new class TuyaTripleSwitch_GP and add new device _TZE200_2hf7x9n3 
Reported in #1597

### CODE REVIEW

**Comment by javicalle** [comment]
Add the `GreenPowerProxy` like in
https://github.com/zigpy/zha-device-handlers/issues/1597#issuecomment-1146210843

File: zhaquirks/tuya/ts0601_switch.py [hunk 1]
```diff
@@ -372,3 +372,77 @@ class TuyaSextupleSwitchTO(TuyaSwitch):
             },
         }
     }
+
+
+## Reported in #1597
```
Comment (javicalle) [inline zhaquirks/tuya/ts0601_switch.py:377]: Remove one `#`

**Review by javicalle — CHANGES_REQUESTED** [review]

**Review by javicalle — CHANGES_REQUESTED** [review]
Add the GreenPowerProxy import like in
https://github.com/zigpy/zha-device-handlers/issues/1597#issuecomment-1146210843

File: zhaquirks/tuya/ts0601_switch.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 """Tuya DP based switches."""
 from zigpy.profiles import zha
-from zigpy.zcl.clusters.general import Basic, Groups, Ota, Scenes, Time
+from zigpy.zcl.clusters.general import Basic, GreenPowerProxy,Groups, Ota, Scenes, Time
```
Comment (javicalle) [inline zhaquirks/tuya/ts0601_switch.py:3]: Add a whitespace:
```
GreenPowerProxy, Groups
```

File: zhaquirks/tuya/ts0601_switch.py [hunk 3]
```diff
@@ -11,6 +11,7 @@
     PROFILE_ID,
 )
 from zhaquirks.tuya import TuyaSwitch
+from zhaquirks.tuya.mcu import MoesSwitchManufCluster, TuyaOnOff, TuyaOnOffManufCluster
```
Comment (javicalle) [inline zhaquirks/tuya/ts0601_switch.py:14]: Remove this line (already exists)

**Review by javicalle — CHANGES_REQUESTED** [review]
Just the requested changes.

**Comment by javicalle** [comment]
I'm sorry that this is giving you more trouble than expected.
I don't know if you know that in the 'Files changed' tab you can see the changes you are introducing in your PR:
* https://github.com/zigpy/zha-device-handlers/pull/1601/files

The latest changes have removed an existing class, which is not the point of your PR.

Since it was also necessary to add the 2 switch device definition, I have taken the liberty of creating a new PR (#1604) with the definitions for the 1 to 4 switch devices. I hope you don't mind.

If it is fine to you, please, close this PR and let's continous with the new one.

**Comment by caslubas** [comment]
> If it is fine to you, please, close this PR and let's continous with the new one.

No problem for me, I'm still learning how to work with github.

**Pull request closed without merging**